common/testing/Makefile.mk

```diff
@@ -6,5 +6,6 @@ noinst_LTLIBRARIES += common/testing/libolatesting.la \
 common_testing_libolatesting_la_SOURCES = \
     common/testing/MockUDPSocket.cpp \
     common/testing/TestUtils.cpp
+common_testing_libolatesting_la_CXXFLAGS = $(COMMON_TESTING_PROTOBUF_FLAGS)
```

Comment:
What was the exact message you were getting for this please and I'll see if we can use a more bespoke and appropriate set of flags?

Reply:
It's on a big machine and I will start it asap, say in a week or so.